src/extension.ts

```diff
@@ -110,4 +113,89 @@ export const activate = () => {
         if (!getExtensionSetting('runOnSave') || reason === vscode.TextDocumentSaveReason.AfterDelay) return
         waitUntil(performFixes())
     })
+
+    vscode.workspace.onDidChangeTextDocument(
+        ({ contentChanges, document }) => {
+            if (!getExtensionSetting("insertMissingCommaOnEnter")) {
+                return;
+            }
+
+            if (
+                document.languageId !== "json" &&
+                document.languageId !== "jsonc"
+            ) {
+                return;
+            }
+
+            if (contentChanges.length === 0) {
+                return;
+            }
+
+            const editor = vscode.window.activeTextEditor;
+
+            if (
+                document.uri !== editor?.document.uri ||
+                ["output"].includes(editor.document.uri.scheme)
+            ) {
+                return;
+            }
+            if (
+                vscode.workspace.fs.isWritableFileSystem(
+                    document.uri.scheme
+                ) === false
+            ) {
+                return;
+            }
+
+            editor.edit((edit) => {
+                contentChanges.forEach(async (content) => {
```

Comment:
```suggestion
                contentChanges.forEach(async (change) => {
```

I know its just a naming, but current name `content` can lead to misunderstanding (it doesn't hold any actual content and its not `string` type)